server/test/lib/scheduler/Assignment.spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as chai from "chai";
+const { expect } = chai;
+import { Assignment } from "../../../lib/scheduler/Assignment";
+
+import { USER_ID, WORKSPACE_ID } from "./utils";
+
+describe("Assignment class", () => {
+  let assignment;
+  beforeEach(() => {
+    assignment = new Assignment({
+      userId: USER_ID,
+      workspaceId: WORKSPACE_ID,
+    });
+  });
+
+  it("has tests that work", () => {
+    expect(true).to.equal(true);
+  });
```

Comment:
How strongly do you feel about having this in every test file?

Reply:
I can take them all out. They were useful when setting everything up but I agree that we don't need them now.